pybiopax/api.py

```diff
@@ -91,37 +98,40 @@ def model_from_pc_query(kind, source, target=None, **query_params):
 
     Returns
     -------
-    pybiopax.biopax.BioPaxModel
+    :
         A BioPAX Model obtained from the results of the Pathway Commons query.
     """
     owl_str = graph_query(kind, source, target=target, **query_params)
+    if owl_str is None:
+        # TODO should this just return None or is an error appropriate?
```

Comment:
@bgyori check this behavior

Reply:
Interesting, I think it depends if owl_str being None indicates an empty result or if it indicates a technical error in the query call. If the latter, raising the error here or in the `graph_query` function makes sense. For the empty result case, returning None is better.

Reply:
I think an empty result should still be a BioPaxModel, but just with an empty objects dictionary.